### ISSUE TITLE
Add support for shinylive

### ISSUE THREAD

**Issue by fifteenforty** [StevenLShafer/stanpumpR#126]
As the documentation reads: "Shinylive allows you to run Shiny applications entirely in a web browser, without the need for a separate server"

stanpumpR should be faster, more responsive and more reliable when run directly in the browser. As it has no external datasource dependencies, this is the ideal way to run it!

https://github.com/posit-dev/r-shinylive

**Comment by StevenLShafer** [comment]
Wow, you have my interest!

Steve

From: Alexander Clarke ***@***.***>
Sent: Thursday, September 14, 2023 1:45 PM
To: StevenLShafer/stanpumpR ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [StevenLShafer/stanpumpR] Add support for shinylive (Issue #126)


As the documentation reads: "Shinylive allows you to run Shiny applications entirely in a web browser, without the need for a separate server"

stanpumpR should be faster, more responsive and more reliable when run directly in the browser. As it has no external datasource dependencies, this is the ideal way to run it!

https://github.com/posit-dev/r-shinylive

**Comment by fifteenforty** [comment]
Unfortunately this is not yet achievable, due to dependencies on packages that aren't compatible yet with webR. Soon I hope. The list of supported packages is available here: https://repo.r-wasm.org

**Comment by fifteenforty** [comment]
> The stanpumpR PK engine itself has no fundamental dependencies and that subset of the R source code runs happily directly in the WebR environment.

Agreed, I just haven't been able to put the time into figuring out exactly is stopping execution of the whole package. There were limited debugging tools available when I made this comment. Great work on posiTIVA!
